cosmos/profiles/base.py

```diff
@@ -24,6 +24,31 @@
 logger = get_logger(__name__)
 
 
+@dataclasses.dataclass
+class DbtProfileConfigVars:
+    send_anonymous_usage_stats: Optional[bool] = False
+    partial_parse: Optional[bool] = None
+    use_experimental_parser: Optional[bool] = None
+    static_parser: Optional[bool] = None
+    printer_width: Optional[bool] = None
+    write_json: Optional[bool] = None
+    warn_error: Optional[bool] = None
+    warn_error_options: Optional[Dict[Literal["include", "exclude"], Any]] = None
+    log_format: Optional[Literal["text", "json", "default"]] = None
+    debug: Optional[bool] = None
+    version_check: Optional[bool] = None
+
+    def as_dict(self) -> dict[str, Any] | None:
+        result = {
+            field.name: getattr(self, field.name)
+            for field in self.__dataclass_fields__.values()
+            if getattr(self, field.name) is not None
+        }
+        if result != {}:
+            return result
+        return None
```

Comment:
Can you add a test for returning `None` here as identified by Codecov

Reply:
add)